backend-lib/Cargo.lock

```diff
@@ -788,16 +788,18 @@ checksum = "af150ab688ff2122fcef229be89cb50dd66af9e01a4ff320cc137eecc9bacc38"
 
 [[package]]
 name = "jni"
-version = "0.20.0"
+version = "0.21.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "039022cdf4d7b1cf548d31f60ae783138e5fd42013f6271049d7df7afadef96c"
+checksum = "1a87aa2bb7d2af34197c04845522473242e1aa17c12f4935d5856491a7fb8c97"
 dependencies = [
  "cesu8",
+ "cfg-if",
  "combine",
  "jni-sys",
  "log",
  "thiserror",
  "walkdir",
+ "windows-sys 0.45.0",
```

Comment:
This is kind-of annoying. Oh well.

Reply:
They have already bumped to `windows-sys 0.48` in their development branch, so this will likely be resolved in the next release (unless they take the opportunity to jump to `windows-sys 0.52` in which case we'll have the reverse issue).

Reply:
Actually I've just checked, and all of our other dependencies will switch to `windows-sys 0.52` with a `cargo update`. So `jni` will hopefully merge https://github.com/jni-rs/jni-rs/pull/513 before their next release.